pkg/ccl/streamingccl/streamproducer/event_stream.go

```diff
@@ -472,6 +472,7 @@ func (s *eventStream) streamLoop(ctx context.Context, frontier *span.Frontier) e
 					return err
 				}
 			default:
+				// TODO(erikgrinaker): Handle DeleteRange events (MVCC range tombstones).
```

Comment:
We would want to forward those, right? Do we need to trim it? Perhaps file an issue to describe how these these should be handled.  Pr

Reply:
I don't believe we need to handle them in CDC currently, because we'll only drop these across offline spans. They're trimmed to the registration bounds. For C2C, they'll need to be ingested into the target cluster.

If we're going to use these in online spans, we'd probably want to emit them as distinct events over CDC. Alternatively, we could synthesize point deletion events for any deleted points, but a single range deletion can expand into an unbounded number of point events, which seems bad.